DOCKERFILE

```diff
@@ -0,0 +1,57 @@
+FROM python:3.7-stretch
+
+MAINTAINER Gilles Bodart <gillesbodart@users.noreply.github.com>
+ 
+RUN apt-get -qq -y update
+RUN apt-get -qq -y upgrade
+RUN apt-get -qq -y install \
+        gcc \
+        g++ \
+        wget \
+        curl \
+        git \
+        make \
+        sudo \
+        bash-completion \
+        tree \
+        vim
+
+# Use C.UTF-8 locale to avoid issues with ASCII encoding
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY ./docker /app
+
+
+# Install any needed packages specified in requirements.txt
+RUN pip install --trusted-host pypi.python.org -r requirements.txt --verbose
```

Comment:
requirements does not exist

Reply:
Thanks for the remarks indeed I forgot to add my folder